src/legacy/timer.go

```diff
@@ -0,0 +1,84 @@
+package legacy
+
+/*
+#include "defs.h"
+#include "GAME2_2.h"
+*/
+import "C"
+
+type Timer struct {
+	Flags          uint32 // bit 1: raw or interp, bit 2: marked when updated
+	Current        uint32 // current value
+	Target         uint32 // target value
+	Delta_per_tick uint32 // (interp only) the amount to add per tick
+	Max_tick_delta uint64 // (interp only) the maximum delta per tick
+	Last_updated   uint64 // the last tick the value was updated
```

Comment:
```suggestion
	DeltaPerTick uint32 // (interp only) the amount to add per tick
	MaxTickDelta uint64 // (interp only) the maximum delta per tick
	LastUpdated   uint64 // the last tick the value was updated
```
(Go coding convention)